docs/troubleshooting.md

```diff
@@ -190,3 +225,194 @@ __Troubleshooting__:
 If [ExternalDNS is updating DNS zone records](#externaldns-is-not-updating-dns-zone-records) and the [DNS glue records are configured](#dns-glue-records-are-not-configured) correctly, you need to wait for the DNS propagation to complete.
 
 [whatsmydns.net](https://www.whatsmydns.net) is a DNS propagation checker that can give you a rough idea of the DNS propagation status of your DNS records.
+
+## Troubleshooting BKPR Ingress
+
+### Let's Encrypt
+
+The inability to access a Kubernetes Ingress resource over HTTP/S is likely caused by one the following scenarios:
+
+1. Ingress resource lacks the necessary annotations
+1. Domain self-check fails
+1. Let's Encrypt rate-limiting
+1. Invalid email address (MX records for email domain did not exist)
+
+In the next sections we will describe the troubleshooting steps required to identify the underlying problem and how to fix it.
+
+__Accessing `cert-manager` logs__:
+
+`cert-manager` logs can be retrieved directly from the pod:
```

Comment:
This sentence should end with a full stop, not a colon

Reply:
Fixed.